### ISSUE TITLE
Homepage-Policy isn't working

### ISSUE THREAD

**Issue by Lukas0000** [mozilla/policy-templates#140]
I am using Firefox ESR on Windows 7 and all policies are working. With one exception, the homepage I set isn't loaded at startup.
In the rsop.msc the homepage is not shown, as you can see on the left side in the picture, on the right side you can see the policy configuration.
![policy](https://user-images.githubusercontent.com/39618791/40655498-cefddfec-6341-11e8-9fa2-7f4e34228340.jpg)

I already replaced the policy templates in the central store.

The problem is appearing on all of our devices

**Comment by ITHand87** [comment]
Hello, I have the same problem on our side. Windows 10 with Firefox ESR 60.0.0.1. 
This parameters don't seem work too:
- Override the first run  page
- Override the upgrade page
- URL for Home page

But, the "about: config" and "about:profile" and some others parameters seems to be applied correctly. 

Tested on multiple computers and differentes configurations.

**Comment by ITHand87** [comment]
Indeed, it seems to work with the "http://www.". It's strange, because when you manually enter the value for the home page without, it works ...
Thank you!
To make a configuration suggestion, where can it be done?

**Comment by malti84** [comment]
Hello, i have the same Problem. I can't set the Startpage/Homepage via GPO. Tested with this URL's:
* https://google.ch
* https://www.google.ch
* www.google.ch
System: Windows 10 / 1709, Firefox Quantum 60.0.1 / Files GPO last (21days Old)

Proxy, Bookmarks, etc. works fine for me. I also set it with Computer GPO's, same!
in the Registry of the Computer, i see the value from the Homepage

![2018-05-30 09_06_21-mremoteng - confcons xml - 10 0 10 20-dcakv](https://user-images.githubusercontent.com/39758220/40704592-d05953d8-63e8-11e8-9418-69508890a965.png)

Thanks for help.
Andreas

**Comment by mkaply** [comment]
Our plan is to move homepage to a machine GPO so that we can allow it on the ESR.
